### ISSUE TITLE
When connected to HDMI, audio comes from the device

### ISSUE THREAD

**Issue by jgranick** [blackberry/SDL#32]
Hi!

I discovered today that when the device is connected to a TV using micro-HDMI, audio plays from the device, when it should play only on the TV.

Do you know why this occurs? This appears to impact SDL games (Frogatto, NME-based, etc) but not other applications on the OS

**Comment by jnicholl** [comment]
Yes, I know why this occurs. It's not easy to fix, however, since it relies on having the audio driver have a separate event loop to listen for audiodevice events and switch the audio output device when required.
I have been told that a fix in the underlying system is planned but I do not know when that will occur.
If this is becoming a serious issue for you, I have both a hacky workaround or could explain how to implement the full solution.
Thanks,
Jeremy
